homeassistant/components/roborock/strings.json

```
@@ -428,6 +428,24 @@
     "get_maps": {
       "name": "Get maps",
       "description": "Get the map and room information of your device."
+    },
+    "vacuum_goto": {
+      "name": "Vacuum go to",
+      "description": "Go to the specified coordinates.",
+      "fields": {
+        "x_coord": {
+          "name": "X coordinate",
+          "description": "X-coordinate."
```

```suggestion
          "name": "X-coordinate",
          "description": ""
```
Same for Y, we can leave out the description as I think translating the same thing twice seems very unnecessary.